docs/docs/concepts/configuration.md

```diff
@@ -1,42 +1,58 @@
 ---
 title: Configuration
 ---
+
 When generating a new Sequent project the configuration can be found in `config/initializers/sequent.rb`.
 You are free to determine your own location as long as you require the file at startup of your application.
 
-There are many configuration options, but generally sticking to the defaults should be sufficient. With that thought in mind this
-chapter is divided into 3 sections:
+There are many configuration options, but generally sticking to the defaults should be sufficient. With that thought in
+mind this chapter is divided into 3 sections:
 
 1. The [minimum configuration](#minimum-configuration) you need
 2. [Configuration options](#frequently-used-configuration) you will likely want to change
 3. The [complete overview](#complete-configuration) of options. Basically the rest.
 
-
 ## Minimum configuration
 
-
 As a bare minimum you need:
 
 ```ruby
 require './db/migrations'
 
 Sequent.configure do |config|
- config.migrations_class_name = 'Migrations'
+  config.migrations_class_name = 'Migrations'
 
- config.command_handlers = [
-   YourCommandHandler.new,
-   OtherCommandHandler.new,
- ]
+  # sequent >= 6.0.2
+  config.enable_autoregistration = true
+
+  # sequent < 6.0.2
+  config.command_handlers = [
+    YourCommandHandler.new,
+    OtherCommandHandler.new,
+  ]
+
+  config.event_handlers = [
+    MyProjector.new,
+    MyWorkflow.new,
+  ]
+  # end sequent < 6.0.2
 
- config.event_handlers = [
-   MyProjector.new,
-   MyWorkflow.new,
- ]
 end
 ```
 
-The `migration_class_name` is the name of the class used to define your Migrations. See the [Migrations](migrations.html) chapter for an in-depth explanation.
+The `migration_class_name` is the name of the class used to define your Migrations. See
+the [Migrations](migrations.html) chapter for an in-depth explanation.
 
+Sequent 6.0.2 introduced autoregistration of `command_handlers` and `event_handlers` via
+setting `enable_autoregistration` to `true`.
+Autoregistered classes will be appended to any already manually registered `command_handlers` and `event_handlers`. If
+Sequent detects duplicates it will currently fail.
+When setting `enable_autoregistration` to `true` in your `initializer`
+any [CommandHandlers](command-handler.html), [Projectors](projector.html) and [Workflows](workflow.html) are
+automatically registered
+in your Sequent configuration. When you have base classes that you don't want to register your self you can
+set `self.abstract_class = true` for these classes. Another option to skip autoregistration is to set
```

Comment:
"When you have base classes that you don't want to register your self..." should probably say: "When you have base classes that you don't want to have automatically registered..."